src/admin/components/forms/field-types/Relationship/types.ts

```diff
@@ -10,7 +10,7 @@ export type Props = Omit<RelationshipField, 'type'> & {
 
 export type Option = {
   label: string
-  value: string | number
+  value: string
```

Comment:
Not having it as a string would break sortable react-selects. I added number back and fixed it by stringifying the value in react-select using `''+value`, though. Added it back, should be fine now!